settings.json

```diff
@@ -0,0 +1,600 @@
+{
+    "type": "object",
+    "properties": {
+        "ALLOWED_HOSTS": {
+            "type": "array",
```

Comment:
It's an array of strings specifically, right? Should be able to specify that in the schema.